capabilities/devops.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: capability
+
+asset-name: devops
+
+description: "DevOps is the ability to do multiple operations simultaneously, improving efficiency and speed of
+deployment. See how we do it."
+
+tags: [DevOps]
+
+title: DevOps
+
+subtitle: "It’s developers and operations working together to safely (but quickly) get your code out to your servers."
+
+text: '<p>We do DevOps for all our clients. </p>
+<p>Traditionally, developers write code, operations deploys it. But what happens in a siloed if something goes wrong?
```

Comment:
Add the word, environment, after:
But what happens in a siloed

Reply:
Yes, thanks!